mkosi/config.py

```diff
@@ -2015,6 +2016,15 @@ def parse_ini(path: Path, only_sections: Collection[str] = ()) -> Iterator[tuple
         parse=config_make_list_parser(delimiter=",", parse=make_path_parser(required=False)),
         help="Add a user-provided initrd to image",
     ),
+    ConfigSetting(
+        dest="initrd_hostonly_microcode",
```

Comment:
`MicrocodeHost=` or maybe `MicrocodeHostonly=`? I would leave any mention of the initrd out of the setting name.

Reply:
Went with `MicrocodeHostonly=`.